pkg/lightning/backend/external/onefile_writer.go

```diff
@@ -92,7 +106,26 @@ func (w *OneFileWriter) initWriter(ctx context.Context, partSize int64) (
 		err = w.dataWriter.Close(ctx)
 		return err
```

Comment:
should we return the original error like L122

Reply:
ok, another bug😓，can you create an issue, we need to fix it on master